examples/with-typescript/tsconfig.json

```diff
@@ -1,19 +1,25 @@
 {
   "compilerOptions": {
-    "target": "esnext",
-    "module": "esnext",
-    "jsx": "preserve",
-    "lib": ["dom", "es2017"],
-    "moduleResolution": "node",
-    "allowJs": true,
-    "noEmit": true,
-    "strict": true,
-    "allowSyntheticDefaultImports": true,
-    "skipLibCheck": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true,
-    "removeComments": false,
-    "preserveConstEnums": true,
-    "sourceMap": true
+    "allowJs": true, 
+    "jsx": "preserve", 
+    "target": "esnext", 
+    "module": "esnext", 
+    "lib": ["dom", "es2017"], 
+    "noEmit": true, 
+    "moduleResolution": "node", 
+    "allowSyntheticDefaultImports": true, 
+    "isolatedModules": true, 
+    "esModuleInterop": true, 
+    "strict": true ,
+    "noImplicitAny": true ,
+    "strictNullChecks": true ,
+    "strictFunctionTypes": true, 
+    "strictBindCallApply": true ,
+    "strictPropertyInitialization": true ,
+    "noImplicitThis": true ,
+    "alwaysStrict": true, 
```

Comment:
As mentioned [here](https://github.com/zeit/next.js/pull/7169#discussion_r280717188), `"strict": true` automatically sets the configs in lines 14-20 to `true`, so it might be better to remove them to declutter the tsconfig.